07_classes.md

```diff
@@ -0,0 +1,565 @@
+---
+title: "Chapter 7: Classes"
+description: How to define your own types and why. 
+---
+
+# Chapter 7: Packages, Modules and Imports
+
+## Chapter objectives
+
+Classes are a very important concept for writing a full-fledged application in Python rather than just a little script because they will help keep your code clean and robust.
```

Comment:
Maybe?
```suggestion
Classes are a very important concept for writing a full-fledged application in Python because they will help keep your code clean and robust.
```